.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,77 @@
+name: Build for Github Pages
+
+on:
+  push:
+    branches: dev
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    environment: dev
```

Comment:
I think the GitHub action will create a `github-pages` one from a quick skim of the docs